composer.json

```diff
@@ -18,29 +18,28 @@
         }
     ],
     "require": {
-        "php": "^7.2",
         "composer/package-versions-deprecated": "^1.11",
-        "danskernesdigitalebibliotek/ddb-stats-laravel": "^1.0",
+        "danskernesdigitalebibliotek/ddb-stats-laravel": "dev-upgrading-to-lumen-8",
         "danskernesdigitalebibliotek/oauth2-adgangsplatformen": "^1.0",
-        "danskernesdigitalebibliotek/openplatform-client": "^1.0",
-        "laravel/lumen-framework": "^6.0",
-        "softonic/laravel-psr15-bridge": "^1.1",
-        "spatie/laravel-cors": "^1.6"
+        "danskernesdigitalebibliotek/openplatform-client": "dev-upgrading-to-lumen-8",
```

Comment:
Remember to update these to new versions once these exist in the corresponding repositories.

Reply:
Yep :)
From the note section:
_"When the review is done on this PR the references to the the two packages mentioned above will be pointed at their new versions."_